packages/GA4Client/src/initialization.js

```diff
@@ -74,4 +79,12 @@ var initialization = {
     },
 };
 
+function setClientId(clientId, moduleId) {
+    var GA4CLIENTID = 'client_id';
+    var ga4IntegrationAttributes = {};
+    ga4IntegrationAttributes[GA4CLIENTID] = clientId;
+    mParticle.setIntegrationAttribute(moduleId, ga4IntegrationAttributes);
+    mParticle._setIntegrationDelay(moduleId, false);
```

Comment:
Here we set the integration delay to false, but we actually need to set it to true immediately after the `initForwarder` starts.  See https://github.com/mparticle-integrations/mparticle-javascript-integration-google-analytics-4/blob/master/packages/GA4Server/src/GoogleAnalytics4EventForwarderServerSide.js#L30